industry_restaurant/data/appointment_type_resource.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
This file could be merged with the other one if you define the floors/tables first in the manifest

Reply:
So this is created dynamically and if we keep it on top in manifest then it will not work as it not first enabled from the res config settings